GVRf/Framework/framework/src/main/java/org/gearvrf/GVRMeshMorph.java

```diff
@@ -253,6 +250,7 @@ public void setBlendShape(int index, GVRSceneObject obj)
     public void setBlendShape(int index, GVRVertexBuffer vbuf)
     {
         int shapeDescriptorFlags = 0;
+        float[] vec3dataBase;
```

Comment:
perhaps i'm going blind, where is this vec3dataBase being used in this method?

Reply:
I took this out

Reply:
vec3dataBase seems to be still here.